modules/templates/helper.go

```diff
@@ -859,12 +859,10 @@ func RenderLabel(label *issues_model.Label) string {
 
 	return fmt.Sprintf("<span class='ui label scope-parent' title='%s'>"+
 		"<div class='ui label scope-left' style='color: %s !important; background-color: %s !important'>%s</div>"+
-		"<div class='ui label scope-middle' style='background: linear-gradient(-80deg, %s 48%%, %s 52%% 0%%);'>&nbsp;</div>"+
```

Comment:
The benefit of removing it is that the HTML becomes simpler 😁 

And if we want to keep it ..... I think it needs `aria-hidden`, otherwise it might be not friendly to screen readers.

Personally I prefer removing it a little.